engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/semantic/ImportExportTest.scala

```diff
@@ -81,10 +81,10 @@ class ImportExportTest
   }
 
   after {
-    ctx.close()
-    ctx = null
     ProjectUtils.deleteRecursively(tmpDir)
     pkg = null
+    ctx.close()
+    ctx = null
```

Comment:
Is this correct? Wouldn't this be a problem on windows if ctx keeps a ref to some path in `tmpDir`?